components/fl-orchestrator/Makefile

```
@@ -33,11 +36,19 @@ install:
 .PHONY: install
 
 clean:
+	rm -rf bin
 	rm -rf dist
 .PHONY: clean
 
-protobuf:
-	protoc --go_out=. --go_opt=paths=source_relative \
-		--go-grpc_out=. --go-grpc_opt=paths=source_relative \
-		pkg/proto/fl_orchestrator.proto
-.PHONY: protobuf
+bin/protoc:
+	./scripts/download-protoc.sh $(PROTOC_VERSION)
+
+bin/protoc-gen-go:
+	GOBIN="$(GOBIN)" go install google.golang.org/protobuf/cmd/protoc-gen-go
```

I don't think you need this `GOBIN="$(GOBIN)"`, it is already getting assigned at the top of the file:

```suggestion
	go install google.golang.org/protobuf/cmd/protoc-gen-go
```

---

I need this, because `GOBIN` defined at the top of the file is a variable for the Makefile, while I need a variable in the shell here. That's why I'm assigning the shell variable `GOBIN` with the value of the Makefile variable `GOBIN`.

---

Good point.